packages/expo/metro.config.js

```diff
@@ -1,3 +1,28 @@
 const {createMetroConfiguration} = require('expo-yarn-workspaces');
 
-module.exports = createMetroConfiguration(__dirname);
+const config = createMetroConfiguration(__dirname);
+
+config.server = {
+  enhanceMiddleware: middleware => {
```

Comment:
What do you think about moving this to a function at the top? would be easier to read.

Reply:
Sure! Will do in the next PR! Thanks!